contributing.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to this Repo
+## What should I work on?
+- Take a look at our open issues, if you are new to this project, look for the ones with the
+`Good First Issue` tag.
+- Take a look at our projects, if you think there is something you would like
+to work on that contributes to the project, then feel free to work on it and make a PR!
+- You can always reach out to me ([hello@czhang.me](mailto:hello@czhang.me)) or any of the
+contributors, we can talk about what our next steps are.
+
+## Development Process
+1. If you have something you would like to work on, or you found a bug, please open an issue
+if there isn't one already. If the issue is releated to a certain project, please
+tag the project in the issue.
+1. If you would like to "own" a certain open issue, assign yourself to it.
+1. Do your work on a feature branch (preferably on your own fork), once you are done,
+make a PR! Draft PRs are always welcome so that any of the contributors can give you early
+feedbabck.
```

Comment:
feedbabck -> feedback

this is not the code review i'm used to

Reply:
maybe i should've read this, it's 2am cut me some slack 😂